lib/kafka_ex/default_server.ex

```diff
@@ -0,0 +1,309 @@
+defmodule KafkaEx.DefaultServer do
```

Comment:
No there is a [0.8](https://github.com/kafkaex/kafka_ex/pull/125/files?diff=split#diff-3) server